src/components/RightSidebar/RightSidebar.vue

```diff
@@ -71,6 +71,12 @@
 			<MatterbridgeSettings
 				v-if="canModerate && matterbridgeEnabled" />
 		</AppSidebarTab>
+		<Description
```

Comment:
shouldn't this markup appear at the top somehow ?

Reply:
What do you mean @PVince81?

Reply:
oh yeah sure!

Reply:
done!